documentation/working-with-testcafe-studio/recording-tests/test-actions/browser-actions.md

```diff
@@ -19,21 +19,21 @@ Resizes the browser window to the specified dimensions.
 
 Parameter | Type  | Description
 --------- | ----- | -------------
-*Width*  | Number | The new width, in pixels.
-*Height* | Number | The new height, in pixels.
+*Width*  | Number | The width (in pixels).
```

Comment:
@dirk-pieterse 
Why you delete "new" from "The new width, in pixels."?  And why you don't delete "new" from the Height parameter's description?

This is such a description of the Resize method in MSDN: https://msdn.microsoft.com/en-us/library/system.windows.automation.provider.itransformprovider.resize(v=vs.110).aspx

Reply:
Because it isn't necessary to add 'new'
The width (in pixels)
The height (in pixels)